burn-compute/tests/dummy/tune/benchmark.rs

```diff
@@ -0,0 +1,48 @@
+use std::marker::PhantomData;
+
+use burn_compute::{server::Handle, tune::AutotuneKernel};
+use derive_new::new;
+
+use crate::dummy::{DummyClient, DummyKernel, DummyServer};
+
+#[derive(new)]
+pub struct DummyBenchmark<O> {
+    client: DummyClient,
+    kernel_constructor: Box<dyn Fn() -> Box<dyn DummyKernel>>,
+    _operation: PhantomData<O>,
+}
+
+impl<'a, O: AutotuneKernel<DummyServer>> TuneBenchmark<O, DummyServer> for DummyBenchmark<O> {
+    fn make_kernel(&self) -> Box<dyn DummyKernel> {
+        (self.kernel_constructor)()
+    }
+
+    fn execute(&self, kernel: Box<dyn DummyKernel>, handles: &[&Handle<DummyServer>]) {
+        self.client.execute(kernel, handles);
+    }
+
+    fn sync(&self) {
+        self.client.sync()
+    }
+}
+
```

Comment:
Is it still used?

Reply:
No, must have been brought back from the dead somehow when merging
Also, @nathanielsimard can you help out with the CI, I don't understand the where the no-std error comes from